Acceleratio.XCellKit/ExcelExport/Data/ExportData.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data;
+using System.IO;
+
+namespace Acceleratio.XCellKit.ExcelExport
+{
+    public class ExportData
+    {
+        public static Stream OpenEntryStreamForWriting()
```

Comment:
SO, XCellKit should not write anything to the disk in this manner. the calling code(which is not in the XcellKit dll) should be the one to create/open the stream and pass it along to this code. XCellKit should not care or know about where the stream actually comes from.